lesson01/assignment/index2.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style2.css">
+    <title>Assignment Lesson 1 | Array Methods</title>
+</head>
+<body>
+    <h1>Array Methods in the Browser</h1>
+    <form id="first-form">
+        <label>Type the first thing that come accross to your mind:</label>
+            <p>"Remember your word must contain at least 5 characters"</p>
+        <input type="text" class="entry">

```

Comment:
Label elements need to be associated with the input they're for, either by wrapping around the input element, or by using the "for" attribute to match the input's id. https://www.w3schools.com/tags/tag_label.asp